javascript/src/reducers/layers.js

```diff
@@ -3,7 +3,8 @@ import {ADD_VARIABLE, REMOVE_VARIABLE} from "../actions/variables"
 import {TOGGLE_VISIBILITY} from "../actions/map"
 import {FINISH_JOB} from "../actions/job"
 import {TOGGLE_LAYER} from '../actions/layers'
-import {RECEIVE_ZONES, RECEIVE_GEOMETRY} from "../actions/zones";
+import config from 'seedsource/config'
+import tilesIndex from 'tilesIndex'
```

Comment:
I added path.resolve('../tiles') to csrt webpack (different repository) since using: import tilesIndex from '../tiles/tilesIndex' above wasn't working. Also, I will commit an empty tilesIndex.json to avoid import errors.

Reply:
Ok. So, tiles are application-specific. Perhaps they could be loaded by the application (e.g., CSRT), and set in `config.js`, then read from there. Spares the need for an extra webpack resolve.